_layouts/default.html

```diff
@@ -1,173 +1,193 @@
+                </li>
+                <li>
+                  <a href="https://github.com/auto-wcag/auto-wcag/">GitHub repository</a>
+                </li>
+              </ul>
+            </li>
 
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown"
-          role="button" aria-haspopup="true" aria-expanded="false">
-            About Auto-WCAG
-          <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li><a href="{{ site.baseurl }}/pages/about.html">
-              About Us
-            </a></li>
-            <li><a href="{{ site.baseurl }}/pages/contribute.html">
-              How to contribute
-            </a></li>
-            <li><a href="https://github.com/auto-wcag/auto-wcag/">GitHub repository</a></li>
           </ul>
-        </li>
-
-      </ul>
-    </div><!--/.nav-collapse -->
-  </div>
-</nav>
-
-<main class="container">
-  {% if page_title %}
-    <h1>{{ page_title }}</h1>
-  {% endif %}
-
-  {{ content }}
-</main>
-
-<footer id="footer"><nav class="container"><div class="row">
-  <div class="col-sm-4">
-    <h2>Rule Creation</h2>
-    <ul>
-      <li><a href="{{ site.baseurl }}/pages/design/rule-design.html">
-        Rule design
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/design/review.html">
-        Review process
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/design/rule-template.html">
-        Rule template
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/design/style-guide.html">
-        Style guide
-      </a></li>
-    </ul>
-
-  </div> <div class="col-sm-4">
-    <h2>Rule Structure</h2>
-    <ul>
-      <li><a href="{{ site.baseurl }}/pages/structure/terminology.html">
-        Terminology
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/structure/test-modes.html">
-        Test modes
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/algorithms/">
-        Algorithms
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/structure/aggregation.html">
-        Result aggregation
-      </a></li>
-      <li><a href="{{ site.baseurl }}/pages/structure/accessibility-support.html">
-        Accessibility support
-      </a></li>
-    </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+    
+    <!-- Begin page content -->
+    <main class="container">
+      {% if page_title %}
+        <div class="page-header">
+          <h1>{{ page_title }}</h1>
+        </div>
+      {% endif %}
+      <p>
+        {{ content }}
+      </p>
+    </main>
+    
+    <script src="{{ site.baseurl }}/assets/js/jquery.min.js"></script>
+    <script src="{{ site.baseurl }}/assets/js/bootstrap.min.js"></script>
+    <script>
+      $(document).ready(function () {
+        // append class names to table
+        $('table').each(function() {
+          $(this).addClass('table table-striped table-hover');
```

Comment:
This seems like a problem that could be solved without JS, no?

Reply:
Yes, can do whilst listening to the Liquid Template Generation, but it is an unnecessary plugin to author. 

I will see what I can do about this.
Will create an issue to solve in the coming days.

Reply:
Have to master overriding Liquid Template Generation, will give it a stab once I learn more on that.